app/views/movies/_social.html.erb

```diff
@@ -1,14 +1,61 @@
 <div class="row w-75" id="social">
-  <div class="col-1 icon-movie"  >
-    <%= fa_icon "list" %>
+  <div class="col-3">
+    <div class="row">
+      <div class="col-5">
+        <div class="scores" id="score">
+          <%= render partial: "rates/score", movie: @movie %>
+        </div>
+      </div>
+      <div class="col-4">
+        <p id="user-score" ><%= t ".user_score" %></p>
+      </div>
+    </div>
   </div>
   <div class="col-1 icon-movie" id="bookmark_movies" >
     <%= render partial: "bookmark_movies/bookmark", movie: @movie %>
   </div>
-  <div class="col-1 icon-movie" id="favoriate-movie" >
+
+  <div class="col-1 icon-movie" id="favoriate-movie">
     <%= render partial: "favoriate_movies/favoriate", movie: @movie %>
   </div>
-  <div class="col-1 icon-movie"  >
-    <%= fa_icon "star" %>
+
+  <div class="col-1 icon-movie-rated" id="rating-movie" >
+    <%= render partial: "rates/rate", movie: @movie %>
+  </div>
+</div>
+
+<div class="row" id="rate-show">
+  <div class="col-4">
+  </div>
+  <div class="col-5">
+    <div class="row rating d-flex justify-content-around" >
```

Comment:
chỗ link to này lặp code quá em ơi
viết helper em nhé

Reply:
Vâng ạ !